examples/motoko_examples/phone-book/README.md

```diff
@@ -1,3 +1,69 @@
 # Phone book
 
-Reimplementation of the [Motoko Phone book example](https://github.com/dfinity/examples/tree/master/motoko/phone-book)
\ No newline at end of file
+This example demonstrates a phone book application that is accessible from your
+web browser.
+
+## Introduction
+
+The application is built from the following Motoko source code files:
+
+*  `index.jsx`, which contains the JavaScript, React, and HTML used to generate
+   the front-end user interface for the application when it is launched in a
+   web browser; and
+
+*  `phone_book.ts`, which contains the actor definition and methods exposed by this
+   canister.
+
+## Prerequisites
+
+Verify the following before running this demo:
+
+*  You have downloaded and installed [Node.js](https://nodejs.org).
+
+*  You have downloaded and installed the [DFINITY Canister
+   SDK](https://sdk.dfinity.org).
+
+*  You have stopped any Internet Computer or other network process that would
+   create a port conflict on 8000.
+
+## Demo
+
+1. Start a local internet computer.
+
+   ```text
+   dfx start
+   ```
+
+1. Open a new terminal window.
+
+1. Reserve an identifier for your canister.
+
+   ```text
+   dfx canister create --all
+   ```
+
+1. Build your front-end.
+
+   ```text
+   npm install
```

Comment:
Same issues as with the superhoes example probably, try to make everything an exact clone as far as possible, use the asset canister